src/blocks/Footer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+export const Footer = () => {
+  return (
+    <footer className='bg-gray-950'>
+        <div>
+            <div>
+                <div>
+                    <ul>
+                        <li>Home</li>
+                        <li>Instruction</li>
+                        <li>About game</li>
+                        <li>Contact</li>
+                    </ul>
+                </div>
+                <div>
```

Comment:
Te dwa `div` są zbędne, jak będziesz cokolwiek tu stylował to `ul` też jest kontenerem więc nie potrzebujemy dodatkowego wrappera, chyba że jakieś style wow będą